features/definitions_transformations_attribute.feature

```diff
@@ -0,0 +1,188 @@
+Feature: Step Arguments Transformations with Attributes
+  In order to follow DRY
+  As a feature writer
+  I need to use transformation functions using PHP attributes
+
+  Background:
+    Given a file named "features/bootstrap/User.php" with:
+      """
+      <?php
+      class User
+      {
+          private $username;
```

Comment:
You are in php 8.1 now, using typed properties would be better :)
using constructors with promoted properties can be possible too.

Reply:
@loic425 most of this code comes from copying existing code but you are right that we should use the opportunity to use modern PHP features when possible, I have updated my PR